Python/Python.sublime-syntax

```diff
@@ -328,29 +328,67 @@ contexts:
       scope: constant.language.python
 
   numbers:
-    - match: \b(?i)(0x\h*)L # py2
+    # https://docs.python.org/3/reference/lexical_analysis.html#numeric-literals
+    # hexadecimal
+    - match: \b(?i)(?:0x\h*)(L) # py2
```

Comment:
If you pay close attention, you can see that there are different scopes assigned for the two matches. The py2 legacy one had an additional `long` segment and this couldn't possibly be done by first pushing into a different context. 

It would also be a lot more verbose overall, imo.